modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -0,0 +1,602 @@
+    std::string url = "https://data.openspaceproject.com/dist/skybrowser/page/";
+
+    const std::string browser = "{"
+        "Identifier = '" + idBrowser + "',"
+        "Type = 'ScreenSpaceSkyBrowser',"
+        "Name = '" + nameBrowser + "',"
+        "Url = '" + url + "',"
+        "FaceCamera = false,"
+        "CartesianPosition = " + ghoul::to_string(positionBrowser) +
+        "}";
+    const std::string target = "{"
+        "Identifier = '" + idTarget + "',"
+        "Type = 'ScreenSpaceSkyTarget',"
+        "Name = '" + nameTarget + "',"
+        "FaceCamera = false,"
+        "}";
+
+    openspace::global::scriptEngine->queueScript(
+        "openspace.addScreenSpaceRenderable(" + browser + ");",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    openspace::global::scriptEngine->queueScript(
+        "openspace.addScreenSpaceRenderable(" + target + ");",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    openspace::global::scriptEngine->queueScript(
+        "openspace.skybrowser.addPairToSkyBrowserModule('" + idTarget + "','"
+        + idBrowser + "');",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    openspace::global::scriptEngine->queueScript(
+        "openspace.skybrowser.setSelectedBrowser('" + idBrowser + "');",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    return 0;
+}
+
+int removeTargetBrowserPair(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::removeTargetBrowserPair");
+    std::string id = ghoul::lua::value<std::string>(L, 1);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    TargetBrowserPair* found = module->getPair(id);
+    if (found) {
+        std::string browser = found->browserId();
+        std::string target = found->targetId();
+        found = nullptr;
+
+        module->removeTargetBrowserPair(id);
+
+        // Remove from engine
+        openspace::global::scriptEngine->queueScript(
+            "openspace.removeScreenSpaceRenderable('" + browser + "');",
+            scripting::ScriptEngine::RemoteScripting::Yes
+        );
+
+        openspace::global::scriptEngine->queueScript(
+            "openspace.removeScreenSpaceRenderable('" + target + "');",
+            scripting::ScriptEngine::RemoteScripting::Yes
+        );
+    }
+    
+
+    return 0;
+}
+
+int translateScreenSpaceRenderable(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 5, "lua::translateScreenSpaceRenderable");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    float startX = ghoul::lua::value<float>(L, 1);
+    float startY = ghoul::lua::value<float>(L, 1);
+    float transX = ghoul::lua::value<float>(L, 1);
+    float transY = ghoul::lua::value<float>(L, 1);
+    
+    ScreenSpaceRenderable* renderable = global::renderEngine->screenSpaceRenderable(id);
+    renderable->translate(glm::vec2(transX, transY), glm::vec2(startX, startY));
```

Comment:
This should check that the `renderable` actually exists.  If someone passes an `id` that is not a valid screenspace renderable, it will cause a hard crash